src/pywbem/twisted_client.py

```diff
@@ -388,9 +431,19 @@ def _getResultParams(result):
 
 
 class PullInstances(OpenEnumerateInstances):
-    def __init__(self, creds, namespace, enumeration_context, MaxObjectCount, classname):
+    def __init__(self, creds, namespace, enumeration_context,
+                 MaxObjectCount, classname, **kwargs):
```

Comment:
The same as above [217](https://github.com/zenoss/ZenPacks.zenoss.WBEM/pull/13/files#diff-e76190bcf5fee57f202738cf09872838R217), regarding the signature.